internal/yml/locking.go

```diff
@@ -1,11 +1,12 @@
 package yml
 
 import (
-	"gopkg.in/yaml.v3"
 	"regexp"
+
+	"gopkg.in/yaml.v3"
 )
 
-var lockMarkerRegex = regexp.MustCompile(`(?m)^##\s*lock\s*$`)
+var lockMarkerRegex = regexp.MustCompile(`(?m)^#+\s*lock\s*$`)
```

Comment:
Would rather not support an arbitrary number of hash signs! 😁
```suggestion
var lockMarkerRegex = regexp.MustCompile(`(?m)^##?\s*lock\s*$`)
```
And I would assume that this is just temporary, until we implement the tag and deprecate the comment, because just adding more and more ways to do things will only get people even more confused!

Reply:
Yes, while we support the comment. I think `### lock` should also work.